tests/assertions/does_not_contain.zunit

```diff
@@ -7,6 +7,13 @@
     assert $state equals 0
 }
 
+@test 'Test _zunit_assert_does_not_contain success with empty value' {
+    run assert '' does_not_contain 'foo'
```

Comment:
we could probably kill a lot of repeated functionality with a `assert_not` function